src/workerd/api/node/buffer.h

```diff
@@ -35,20 +36,20 @@ class BufferUtil final: public jsg::Object {
 
   kj::Array<kj::byte> decodeString(jsg::Lock& js,
                                    jsg::JsString string,
-                                   kj::String encoding);
```

Comment:
nit: for this, probably a good idea to include a doc comment referencing the Encoding enum

Reply:
or I can just add something similar to `using EncodingValue = uint8_t;`. wdyt?

Reply:
Yep, that works too